src/Features/Core/Portable/DocumentationComments/AbstractDocumentationCommentFormattingService.cs

```diff
@@ -550,6 +559,13 @@ private static string TrimCrefPrefix(string value)
         private static void AppendTextFromTextNode(FormatterState state, XText element)
         {
             var rawText = element.Value;
+            if ((state.Style & TaggedTextStyle.PreserveWhitespace) == TaggedTextStyle.PreserveWhitespace)
+            {
+                // Don't normalize code from middle. Only trim leading/trailing new lines.
```

Comment:
why trim the leading/trailing?

Reply:
➡️ This implements a fix which a test revealed:
https://github.com/dotnet/roslyn/pull/53140#discussion_r628561538